### ISSUE TITLE
Menu item to edit preferences

### ISSUE THREAD

**Issue by AlekSi** [robcowie/SublimeTODO#40]
Looks like plugin is missing menu items to edit preferences. Example: https://github.com/apophys/SublimeManpage/blob/master/Main.sublime-menu

**Comment by robcowie** [comment]
Yeah. I need to move the preferences from User prefs to a separate plugin preferences file (which is the usual pattern now, though wasn't clear when I first wrote this plugin). I'm struggling to find a way to do that in a backward-compatible way. I need to conditionally migrate settings from `User.Preferences.sublime-settings`.

When I've done that, I can add a menu item
